plugins/by-name/noice/default.nix

```diff
@@ -165,78 +239,93 @@ with lib;
           reverse = true;
         };
       };
-    } "You can add any custom commands that will be available with `:Noice command`";
+    } "You can add any custom commands that will be available with `:Noice` command.";
 
     notify = {
-      enabled = helpers.defaultNullOpts.mkBool true ''
+      enabled = defaultNullOpts.mkBool true ''
         Enable notification handling.
 
         Noice can be used as `vim.notify` so you can route any notification like other messages.
+
         Notification messages have their level and other properties set.
         event is always "notify" and kind can be any log level as a string.
+
         The default routes will forward notifications to nvim-notify.
         Benefit of using Noice for this is the routing and consistent history view.
       '';
-      view = helpers.defaultNullOpts.mkStr "notify" "";
+
+      view = defaultNullOpts.mkStr "notify" "Notify backend to use.";
     };
 
     lsp = {
       progress = {
-        enabled = helpers.defaultNullOpts.mkBool true "enable LSP progress";
+        enabled = defaultNullOpts.mkBool true "Enable LSP progress.";
 
-        format = helpers.defaultNullOpts.mkNullable (with types; either str anything) "lsp_progress" ''
-          Lsp Progress is formatted using the builtins for lsp_progress
+        format = defaultNullOpts.mkNullableWithRaw (with types; either str anything) "lsp_progress" ''
+          Lsp Progress is formatted using the builtins for lsp_progress.
         '';
-        formatDone = helpers.defaultNullOpts.mkNullable (with types; either str anything) "lsp_progress" "";
 
-        throttle = helpers.defaultNullOpts.mkNum (literalExpression "1000 / 30") "frequency to update lsp progress message";
+        format_done =
+          defaultNullOpts.mkNullableWithRaw (with types; either str anything) "lsp_progress_done"
+            ''
+              Lsp Progress is formatted using the builtins for lsp_progress.
+            '';
 
-        view = helpers.defaultNullOpts.mkStr "mini" "";
+        throttle = defaultNullOpts.mkNum (lib.literalExpression "1000 / 30") "Frequency to update lsp progress message.";
```

Comment:
I think this is blamed on me, but I wonder why we're saying `1000 / 30` instead of the simpler `100 / 3`?

Reply:
I thought I saw this in the upstream default so maybe just respecting what they had?

Reply:
Yeah, I guess so. Just seems strange to have the extra `0` given `(100 * N) / (3 * N) = 100 / 3 ≈ 33.33`